packages/core/src/colors.ts

```diff
@@ -77,9 +77,9 @@ const colors = {
     additional: "rgba(220,229,224,0.4)",
   },
 
-  notion: {
-    light: "#DEF1ED",
-    dark: "#3E5860",
+  concept: {
+    light: "#def1ed",
+    dark: "#3e5860",
```

Comment:
I figma så er både #3e5860 og #638b98 brukt (ikon den første og border den andre). Er vi sikre at den første som er riktig?

Reply:
Eg brukte verdiene fra figma basestyles forklaring lys og forklaring tekst. Det er ingen andre ressurstyper som har definert tekstfarge, men når eg sammenligna med designet under Sandbox så synes eg dette såg mest korrekt ut.

Reply:
Det er veldig rart at det har blitt definert 3 farger nå, mens det er vanligvis bare lys og mørk variant. 
![image](https://github.com/NDLANO/frontend-packages/assets/110535103/be84caac-a153-4436-a637-3dab77104e4a)
Det er brukt alle 3 farger i badgen, men som du sier så brukes som regel bare lys og mørk versjoner av farger så derfor lurte jeg på om tekst fargen var den riktige å bruke 🤷‍♂️

Reply:
Ja, ser at designet seier anna farge på ikon og ramme, men slik er det ikkje for nokon andre badges.